provider-utils/awscloudformation/cloudformation-templates/vod-helpers/LambdaFunctions/InputLambda/index.js

```diff
@@ -0,0 +1,84 @@
+
+// Load the AWS SDK for Node.js
+// eslint-disable-next-line import/no-extraneous-dependencies
+/* eslint-disable */
+const AWS = require('aws-sdk');
+/* eslint-enable */
+const jobSettings = require('./settings.json');
+// Set the region
+
+exports.handler = async (event) => {
+  AWS.config.update({ region: event.awsRegion });
+  const s3 = new AWS.S3({ apiVersion: '2006-03-01' });
+  // Object was deleted
+  if (event.Records[0].eventName === 'ObjectRemoved:Delete') {
+    // Grabbing Key name and Bucket name from event
+    const DeletedKey = event.Records[0].s3.object.key;
+    const split = DeletedKey.split('.')[0];
+    const outputBucketName = process.env.OUTPUT_BUCKET;
+
+    // Call S3 to obtain a list of the objects in the bucket
+    const res = await s3.listObjectsV2({
+      Bucket: outputBucketName,
+      Prefix: split,
+    }).promise();
```

Comment:
Do we do pagination for this? What happens if their is more then a 1000 objects listed?